p5js/NeuralNetwork/NeuralNetwork_ImageClassifier_Colors/sketch.js

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Image classification using MobileNet and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+let nn;
+
+function setup() {
+
+  const options = {
+    task: 'imageClassification',
+    inputs: [2,2,4],
+    debug:true
+  }
+  nn = ml5.neuralNetwork(options);
+
+  /**
+   * ///////////////////////////
+   * ADD DATA
+   * ///////////////////////////
+   */
+  addData();
+
+  nn.train({epochs: 100, batchSize:2}, finishedTraining)
+
+}
+
+function finishedTraining(){
+
+  nn.classify([
+    0, 0, 255,255, 0, 0, 255,255,
+    0, 0, 255,255, 0, 0, 255,255
+], gotResults)
+
+}
+
+function gotResults(err, result){
+  if(err){
+    console.error(err)
+  }
+  console.log(result)
+}
+
+function addData() {
+  const myData = [{
+      label: "red-square",
+      value: [
+        255, 0, 0,255, 255, 0, 0, 255,
+        255, 0, 0,255, 255, 0, 0, 255
+      ]
+    },
+    {
+      label: "green-square",
+      value: [
+        0, 255, 0, 255, 0, 255, 0, 255,
+        0, 255, 0, 255, 0, 255, 0, 255
+      ]
+    },
+    {
+      label: "blue-square",
+      value: [
+        0, 0, 255,255, 0, 0, 255,255,
+        0, 0, 255,255, 0, 0, 255,255
+      ]
+    }
+
+  ]
+
+  // method 1
+  myData.forEach(item => {
```

Comment:
p5 examples don't generally use `forEach()` so I might suggesting an old-fashioned `for` loop?